src/Migrations/Version20241101104421.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Abenmada\BackofficePlugin\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Postgres ready!
+ *
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20241101104421 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE SEQUENCE abenmada_backoffice_channel_image_id_seq INCREMENT BY 1 MINVALUE 1 START 1');
```

Comment:
This migration was run even on a mysql database, so an error occurred because some scripts do not work on mysql, there is no control to tell doctrine to run it only on postregress